src/js/components/TaskDetail.js

```diff
@@ -287,37 +353,66 @@ class TaskSidePanelContents extends SidePanelContents {
     super.tabs_handleTabClick(...arguments);
   }
 
+  getNotFound(itemType) {
+    return (
+      <div className="container container-fluid container-pod text-align-center">
+        <h3 className="flush-top text-align-center">
+          {`Error finding ${itemType}`}
+        </h3>
+        <p className="flush">
+          {`Did not find a ${itemType} by the id "${this.props.params.taskID}"`}
+        </p>
+      </div>
+    );
+  }
+
+  getExpandButton() {
+    if (!this.state.showExpandButton) {
+      return null;
+    }
+
+    return (
+      <button
+        className="button button-stroke button-expand"
+        onClick={this.handleExpand}>
```

Comment:
This was part of the `SidePanelContents.js` component which we don't extend anymore, so the functionality is missing.